pci/src/vfio.rs

```diff
@@ -1011,7 +1059,14 @@ impl VfioCommon {
         };
 
         // The config register read comes from the VFIO device itself.
-        self.vfio_wrapper.read_config_dword((reg_idx * 4) as u32) & mask
+        let mut value = self.vfio_wrapper.read_config_dword((reg_idx * 4) as u32) & mask;
+
+        if let Some(cfg) = self.patches.get(&reg_idx) {
+            value &= !cfg.mask;
+            value |= cfg.patch & cfg.mask;
```

Comment:
a.) I'm not sure I would call it `cfg` doesn't seem very descriptive
b.) By `& mask` you're preventing setting bits outside which might be useful for a general approach of just ORing bits.

Reply:
a) Sure I can use `config_patch` instead of `cfg`.
b) It's on purpose that I let nothing get through. The way I implemented it here, I really want to make sure the patch gets applied only for the bits defined by the mask. And at the same time I want to ensure the bits from the mask are completely cleared from the value we retrieved from reading.

Reply:
Since you control `patch` anding it with `mask` protects nothing.

Reply:
Fair enough, I've updated the PR.